test/test_RamsesBlenderExporter.py

```diff
@@ -42,7 +50,8 @@ def test_default_scene_viewer_does_not_crash_blender(self):
 
         for exportable_scene in exporter.get_exportable_scenes():
             inspector = ramses_export.ramses_inspector.RamsesInspector(exportable_scene)
-            inspector.load_viewer()
+            os.chdir(self.addon_path)
```

Comment:
Should not be needed - just pass in the full path to the scene viewer by joining it with "addon_path" here. This way you also don't have to pass in the platform

Reply:
You mean changing the "platform" argument to "scene viewer path" directly?